ios/MissionHub.xcodeproj/xcshareddata/xcschemes/MissionHub.xcscheme

```diff
@@ -60,6 +60,16 @@
             ReferencedContainer = "container:MissionHub.xcodeproj">
          </BuildableReference>
       </BuildableProductRunnable>
+      <CommandLineArguments>
+         <CommandLineArgument
+            argument = "-FIRDebugEnabled"
+            isEnabled = "YES">
+         </CommandLineArgument>
+         <CommandLineArgument
+            argument = "-FIRDebugDisabled"
+            isEnabled = "NO">
+         </CommandLineArgument>
```

Comment:
This seems duplicate but maybe that's how it works.

Reply:
Yeah not sure either on this one. @levieggertcru is this needed for a particular reason?